### PR TITLE
OOS Exec Doc 2024-12-06

### PR DESCRIPTION
**Pull request by blimpa**
Originating forum post: https://forum.sky.money/t/out-of-schedule-executive-proposal-stability-scope-parameter-changes-19-sfs-dsr-ssr-spark-effective-dai-borrow-rate-spark-liquidity-layer/25648

@CivicSage3 , @VoteWizard

### CODE REVIEW

File: governance/votes/Executive vote - December 6, 2024.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+- Increase WBTC-B SF by 4 percentage points from 12.75% to **16.75%**.
+- Increase WBTC-C SF by 4 percentage points from 12.00% to **16.00%**.
+- Increase ALLOCATOR-SPARK-A SF by 2.94 percentage points from 9.31% to **12.25%**.
+
+### Savings Rate Changes
+
+- **Authorization**: [Ecosystem Approval](https://forum.sky.money/t/out-of-schedule-executive-proposal-stability-scope-parameter-changes-19-sfs-dsr-ssr-spark-effective-dai-borrow-rate-spark-liquidity-layer/25648/2)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/out-of-schedule-executive-proposal-stability-scope-parameter-changes-19-sfs-dsr-ssr-spark-effective-dai-borrow-rate-spark-liquidity-layer/25648)
+
+If this executive proposal passes, the then following savings rates changes will take place:
```
Comment (VoteWizard) [inline governance/votes/Executive vote - December 6, 2024.md:55]:
```suggestion
If this executive proposal passes, then the following savings rate changes will take place:
```

File: governance/votes/Executive vote - December 6, 2024.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+---
+
+## Proposal Details
+
+### Stability Fee Changes
+
+- **Authorization**: [Ecosystem Approval](https://forum.sky.money/t/out-of-schedule-executive-proposal-stability-scope-parameter-changes-19-sfs-dsr-ssr-spark-effective-dai-borrow-rate-spark-liquidity-layer/25648/2)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/out-of-schedule-executive-proposal-stability-scope-parameter-changes-19-sfs-dsr-ssr-spark-effective-dai-borrow-rate-spark-liquidity-layer/25648)
+
+If this executive proposal passes, then the [Stability Fees](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.3_Stability_Fee-67e40a3b-f1c2-4dc6-b502-2affeab0b232|57eaf45219bea3b430c2) (SF) for the vault types listed below will be changed as follows:
```
Comment (VoteWizard) [inline governance/votes/Executive vote - December 6, 2024.md:38]:
```suggestion
If this executive proposal passes, then the [Stability Fees (SF)](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.3_Stability_Fee-67e40a3b-f1c2-4dc6-b502-2affeab0b232|57eaf45219bea3b430c2) for the vault types listed below will be changed as follows:
```

For consistency, I’d suggest moving the abbreviation into the link text

**Comment by blimpa** [comment]
# Executive Creation

## Creation

- [x] Sync Community Git
- [x] Ensure you are working from the latest master branch
- [x] Read Associated [Executive Contents sheet](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?pli=1#gid=1593813984)
- [x] Copy the executive template
- [x] Update filename
- [x] Update dates - exact time is not required and should be left at midnight.

- [x] Ensure the title refers to the most impactful/important actions included in the proposal.
- [x] Ensure that the summary refers to all actions included in the proposal.

- [x] Ensure that the publisher (entity publishing the executive on the voting portal) of the executive is listed in the preamble (at the entity level).
- [x] Ensure that the writers of the executive code are listed in the preamble (at the entity level).
- [x] Ensure that the reviewers of the executive code are listed in the preamble (at the entity level).

- [x] Ensure that the office-hours modifier is mentioned as part of the executive copy, if present in the executive spell code.
- [x] Ensure that it is clearly stated which items are affected by the GSM Delay.
- [x] Ensure that the GSM Delay period is clearly communicated.
- [x] Ensure that the proposal expiry is explicitly noted, including the length of time for which the proposal is valid.

## Items

- [x] Add each item to the proposal details section
- [x] Order the items by judged impact/importance. If different from the executive contents, reorder the executive contents sheet so that they match.

- [x] For each item
	- [x] Ensure the executive summary contains a high-level overview of every action contained in the proposal.
	- [x] Ensure that each item or set of items in the executive proposal has a corresponding entry in the proposal details section.
	- [x] Ensure that each item in the copy contains a link to the reasoning for its inclusion.
	- [x] Ensure that each item in the copy contains a link to the authorization for its inclusion.
	- [x] Ensure that each item in the copy contains any links required for voters to determine the context surrounding its inclusion.
	- [x] Ensure that the framing and tense used for each item is future-conditional. Ie: "IF this executive proposal passes, THEN this action will take place."
	- [x] Ensure that hyperlinks to documentation and/or context are added to key parameters, terms, and contracts where reasonable and possible.
	- [x] Ensure that Ethereum and IPFS addresses are hyperlinked such that it is easy to view their details.
	- [x] Ensure that parties implicated in the proposal outcome related to this item are aware of the wording used.

## Self-Review

- [x] Do any '$' parameter signifiers remain? (spell address is unknown at this point)
- [x] Do the executive title, summary, and executive summary accurately reflect executive copy content?
- [x] Read the executive copy in full, with the above checklists and general readability in mind. Make any required changes.
- [x] Run the copy through a spelling and grammar checker - do not blindly accept suggested changes.
- [x] Inform the reviewer that the executive copy is ready to review
- [x] Commit changes, and create a PR to the Community GitHub repository.

**Review by MakerJanSky — CHANGES_REQUESTED** [review]
Nice work, all.

**Comment by MakerJanSky** [comment]
# Executive Document Review

## Content
- [x] Check the title refers to the most impactful/important actions included in the proposal.
- [x] Check that the summary refers to all actions included in the proposal.
- [x] Check that the publisher (entity publishing the executive on the voting portal) of the executive is listed in the preamble (at the entity level).
- [x] Check that the writers of the executive code are listed in the preamble (at the entity level).
- [x] Check that the reviewers of the executive code are listed in the preamble (at the entity level).
- [x] Check that the office-hours modifier is mentioned as part of the executive copy, if present in the executive spell code.
- [x] Check that it is clearly stated which items are affected by the GSM Delay.
- [x] Check that the GSM Delay period is communicated.
- [x] Check that the proposal expiry is explicitly noted, including the length of time for which the proposal is valid.

## Items

- [x] Check each item is present in the proposal details section
- [x] Check items are ordered by judged impact/importance

- [ ] For each item
	- [x] Check the executive summary contains a high-level overview of the item.
	- [x] Check that each item or set of items in the executive proposal has a corresponding entry in the proposal details section.
	- [x] Check that the copy contains a link to the reasoning for its inclusion.
	- [x] Check that the copy contains a link to the authorization for its inclusion.
	- [x] Check that the copy contains any links required for voters to determine the context surrounding its inclusion.
	- [x] Check that the framing and tense used for each item is future-conditional. Ie: "IF this executive proposal passes, THEN this action will take place."
	- [x] Check that hyperlinks to documentation and/or context are added to key parameters, terms, and contracts where reasonable and possible.
	- [x] Check that Ethereum and IPFS addresses are hyperlinked such that it is easy to view their details.
	- [x] Check that parties implicated in the proposal outcome related to this item are aware of the wording used.

## Final Checks

- [x] Read the executive copy in full, with the above checklists and general readability in mind. Raise any issues with the author.
---

**Review by MakerJanSky — APPROVED** [review]
LGTM

**Comment by MakerJanSky** [comment]
Confirming hash for our side:

`0xf3154487b58605fd6398364065ba51165e10514f45d063ed3727de31d2b7a3a7`

**Comment by VoteWizard** [comment]
### Hash of the Document Before Merging

Command:
`cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/b97b2079e2f7ee950676c39f1d67303d19d55fdf/governance/votes/Executive%20vote%20-%20December%206%2C%202024.md' -q -O - 2>/dev/null)"`

Hash:
`0xf3154487b58605fd6398364065ba51165e10514f45d063ed3727de31d2b7a3a7`

The hash matches with the outcome of this online tool:
https://emn178.github.io/online-tools/keccak_256.html

**Pull request merged**